pywinauto/unittests/test_application.py

```diff
@@ -1204,6 +1204,54 @@ def test_find_elements_re(self):
         self.assertTrue(len(windows) >= 1)
 
 
+class ChildWindowSpecificationFromWrapperTests(unittest.TestCase):
+    def setUp(self):
+        """Set some data and ensure the application is in the state we want"""
+        Timings.defaults()
+        self.app = Application().start("Notepad")
```

Comment:
I'd suggest to use `backend="win32"` explicitly as we plan to require it in the future. Also use `_notepad_exe()` instead of just "Notepad".

Reply:
I also changed this in `WindowSpecificationTestCases` where this code was taken from.